layouts/_default/_markup/render-codeblock-mermaid.html

```diff
@@ -0,0 +1,3 @@
+<div class="mermaid">
+  {{- .Inner | safeHTML }}
+</div>
```

Comment:
@deining - you marked this as resolved, but I'm not seeing the change you suggested in the file. Why is that? I'm asking because some of the Mermaid diagrams are failing to render, but if I replace the `<div>` by `<pre>` it works once again.

Reply:
Yeah, this breaks things. See, #1450.